### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by MatthewSteeples** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#703]
"Unless you have assigned a custom policy to the users, they get Global policy, which has disabled enabled by default."

I think that this should read "which has AllowTranscription disabled by default"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 976b8a9b-5a3e-1f3b-b848-d1dfbfd3389d
* Version Independent ID: 2b13a471-3baf-0086-439a-f94abb162fd6
* Content: [Teams cloud meeting recording](https://docs.microsoft.com/en-us/microsoftteams/cloud-recording)
* Content Source: [Teams/cloud-recording.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/cloud-recording.md)
* Service: **msteams**
* GitHub Login: @tonysmit
* Microsoft Alias: **tonysmit**

**Comment by AndreaBarr** [comment]
@MatthewSteeples Thank you for this imput

That is a good rephrasing for this line. I am going to get this to the Teams team so we can get this updated approperatly. Thank you for helping make these docs better. 

I was able to get this into the backlog for the Teams team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
